python/dgl/mock_sparse/sddmm.py

```diff
@@ -0,0 +1,57 @@
+"""Sampled Dense-Dense Matrix Multiplication (SDDMM) operator module."""
+import torch
+
+from .sp_matrix import SparseMatrix
+
+__all__ = ["sddmm"]
+
+
+def sddmm(
+    A: SparseMatrix, mat1: torch.Tensor, mat2: torch.Tensor
+) -> SparseMatrix:
+    r"""Sampled-Dense-Dense Matrix Multiplication (sddmm). `sddmm` multiplies
```

Comment:
- Use uppercase for abbr.
- Add a new line after short description.
- Docstring is in RST not markdown. Use double quote instead of single quote for monofont.

e.g.,

```
r"""Sampled-Dense-Dense Matrix Multiplication (SDDMM).

``sddmm`` multiplies ...
```